docker-compose.yml

```diff
@@ -27,7 +28,6 @@ services:
       - "--dev"
       - "--ws-external"
       - "--rpc-external"
-      - "-lruntime=debug"
```

Comment:
how this docker-compose is used in gitlab or is it just for local?

Reply:
only local

Reply:
I'll think of how to better automate it there.

Reply:
Where? No need to automize anything here. That's just my personal docker-compose file that I'm using for development which I just dropped in here. The '- "-lruntime=debug"' was just a leftover from when we were trying to debug an potential memory issue